.vscode/launch.json

```diff
@@ -0,0 +1,37 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "node",
+      "request": "attach",
+      "name": "Attach debugger"
+    },
+    {
+      "type": "node",
+      "runtimeExecutable": "deno",
+      "request": "launch",
+      "name": "Debug current file",
+      "program": "${file}",
+      "cwd": "${workspaceFolder}"
+      // "env": {
+      //   "PATH": "/path/to/polkadot/folder:${env:PATH}"
+      // }
+    },
+    {
+      "type": "node",
+      "request": "launch",
+      "runtimeExecutable": "deno",
+      "runtimeArgs": ["task", "test", "--inspect-brk"],
+      "name": "Debug current test file",
+      "program": "${file}",
+      "cwd": "${workspaceFolder}",
+      "attachSimplePort": 9229
```

Comment:
If this is not explicitly set, VSCode launches this task with the following arguments
```
/path/to/deno run --inspect-brk=127.0.0.1:53033 --allow-all ./the/current/file.ts
```